common/utils/works.js

```diff
@@ -214,11 +214,25 @@ export function getWorkTypeIcon(work: Work): ?string {
 }
 
 export function getLocationType(work: Work, locationType: string) {
+  // TODO rename?
   const [item] = work.items
     .map(item =>
-      item.locations.find(location => location.locationType.id === 'iiif-image')
+      item.locations.find(location => location.locationType.id === locationType)
     )
     .filter(Boolean);
 
   return item;
 }
+
+type LocationType = 'PhysicalLocation' | 'DigitalLocation';
+
+export function getItemLocationsOfType(work: Work, locationType: LocationType) {
```

Comment:
Could we add a return type to this function, as it's best practise and will make the function name more clear